pkg/scheduler/ugm/manager.go

```diff
@@ -193,20 +193,26 @@ func (m *Manager) DecreaseTrackedResource(queuePath string, applicationID string
 	return nil
 }
 
-func (m *Manager) GetUserResources(user security.UserGroup) *resources.Resource {
-	return nil
+func (m *Manager) GetUserResources(user security.UserGroup) (*resources.Resource, error) {
+	if m.userTrackers[user.User] != nil {
+		return m.userTrackers[user.User].queueTracker.resourceUsage, nil
+	}
+	return nil, fmt.Errorf("user %s is not available in user trackers map", user.User)
 }
 
-func (m *Manager) GetGroupResources(group string) *resources.Resource {
-	return nil
+func (m *Manager) GetGroupResources(group string) (*resources.Resource, error) {
+	if m.groupTrackers[group] != nil {
+		return m.groupTrackers[group].queueTracker.resourceUsage, nil
+	}
+	return nil, fmt.Errorf("group %s is not available in group trackers map", group)
 }
 
-func (m *Manager) GetUsersResources() []*UserTracker {
-	return nil
+func (m *Manager) GetUsersResources() map[string]*UserTracker {
+	return m.userTrackers
```

Comment:
This could cause race conditions. The caller gets that map and can manipulate the map.
maps are not concurrency safe and need accessing under lock. We should pass back a copy of the content.
That is why the `[]*userTracker` was chosen